packages/sit-onyx/src/composables/useClipping.ts

```diff
@@ -0,0 +1,92 @@
+import { useGlobalEventListener } from "@sit-onyx/headless";
+import { computed, nextTick, onMounted, ref, type Ref } from "vue";
+
+export function useClipping(
+  popoverRef: Ref<HTMLElement | null>,
+  popoverWrapperRef: Ref<HTMLElement | null>,
+  popoverPosition: Ref<string>,
+  isVisible: Ref<boolean>,
+  clipping: Ref<boolean>,
+) {
```

Comment:
```suggestion
export type UseClippingOptions = {
  popoverRef: Ref<HTMLElement | null>;
  popoverWrapperRef: Ref<HTMLElement | null>;
  popoverPosition: Ref<AnchorPosition | OpenDirection>;
  isVisible: Ref<boolean>;
  clipping: Ref<boolean>;
};

export function useClipping(options: UseClippingOptions) {
```

Would prefer to use an options object here